test/plugins/individual-collateral/compoundv3/CusdcV3Wrapper.test.ts

```diff
@@ -598,6 +598,18 @@ describeFork('Wrapped CUSDCv3', () => {
       )
     })
 
+    it('regression test: able to claim rewards even when they are big without overflow', async () => {
+      // Nov 28 2023: uint64 math in CusdcV3Wrapper contract results in overflow when COMP rewards are even moderately large
+
+      const compToken = <ERC20Mock>await ethers.getContractAt('ERC20Mock', COMP)
+      expect(await compToken.balanceOf(wcusdcV3.address)).to.equal(0)
+      await advanceTime(1000)
+      await enableRewardsAccrual(cusdcV3, bn('2e18')) // enough to revert on uint64 implementation
```

Comment:
i think we need to push it to 20e18, no?

Reply:
Yes

Reply:
Confirmed the test fails if the change is reverted so seems to be good enough the example.

Reply:
Yes I confirmed the test fails, so I don't see why it would need to be higher

Reply:
yup nvm, this isn't the variable i thought it was.